new/resource-current-value.md

```diff
@@ -0,0 +1,98 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io
+Status: Draft
+Type: Standards Track
+---
+
+# Allow Current Value To Be Read From Resource
```

Comment:
Did you decide to call this `current_resource` instead of `current_value` after writing this title and the paragraph of text under it? I think these might just need to be updated to:

```
# Allow Current Resource to be Read from Resource Collection

Resources get a `current_resource` method, which discovers an existing resource in the resource collection
```